t/mojo/daemon.t

```diff
@@ -109,6 +109,15 @@ subtest 'Transaction' => sub {
 my $app = Mojolicious->new;
 my $ua  = Mojo::UserAgent->new(ioloop => Mojo::IOLoop->singleton);
 
+my $whatever = 'Failed!';
+$app->hook(
+  before_app_start => sub {
+    my ($app) = shift;
+    my $ok = ref $app && $app->isa('Mojolicious');
+    $whatever = $ok ? 'Whatever!' : 'Wrong type';
```

Comment:
Way too complicated again. Why have special case that is now untested?

Reply:
What do you mean "untested"? This is tested indeed... if I change the 'Whatever!' string, the tests are failing!
Or did you mean something I don't get?

Reply:
The `Wrong type` variant appears untested and rather pointless. You could just ref the app and use that as test value.

Reply:
Or better yet, call a method in `$app` and use that, like `->moniker` or so.